varinfo/attribute_container.py

```diff
@@ -100,19 +99,13 @@ def _get_configured_attribute(
         value.
 
         """
-        cf_override = self.cf_config['cf_overrides'].get(attribute_name)
-        cf_supplement = self.cf_config['cf_supplements'].get(attribute_name)
+        cf_override = self.cf_overrides.get(attribute_name)
```

Comment:
What if this function was just:
``` 
    return self.cf_overrides.get(attribute_name, raw_attribute_value)
```

Reply:
Ha, yeah. Much simpler. Clearly I was still too caught up in the prior structure of the function.

Reply:
Updated in [edf9db](https://github.com/nasa/earthdata-varinfo/pull/36/commits/edf9db6e153c193546a7909d76876303c689f0f3).

Reply:
to be fair, I only remembered my last cleanup attempt.